### ISSUE TITLE
All-encompassing Search

### ISSUE THREAD

**Issue by peth-yursick** [MetaFam/TheGame#1621]
### What would you like to be added?

I would like the search functionality to include:
- Patrons
- Paths & Playbooks (the single unified page for all things educational)
- Quests

✅ ~~Additionally, it would be nice if the search dropdown was stylized like the player dropdown.~~

I think this is blocked by "Paths & Playbooks" still not being a single page.
Will that be a new page or is it being merged into /playbooks @luxumbra?

### Why is this needed?

The Search feature, as it currently is, was drafted almost 2 years ago - when patrons & questchains didnt exist yet.
MetaGame has since evolved, so for search to be really useful, it should include these really important parts of MetaGame.

**Comment by luxumbra** [comment]
I'm building it as a separate page '/paths-and-playbooks' currently, updated the mega menu to match what the Figma shows and have left the other pages as they are for reference and deletion once my work is complete.

**Comment by peth-yursick** [comment]
> I'm building it as a separate page '/paths-and-playbooks' currently, updated the mega menu to match what the Figma shows and have left the other pages as they are for reference and deletion once my work is complete.

Gotcha, thank you. Sorry for offtopic but would actually prefer if it stayed labeled as "Learn" in the menu, or even changed to just "Library" on both the menu & the page itself if you want the menu & page labels to stay consistent.. Thoughts?

**Comment by Udit-takkar** [comment]
@luxumbra have you created this page?
